hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/ha/TestSequenceIDGenerator.java

```diff
@@ -70,8 +73,6 @@ public void testSequenceIDGenUponNonRatis() throws Exception {
   @Test
   public void testSequenceIDGenUponRatis() throws Exception {
     OzoneConfiguration conf = SCMTestUtils.getConf();
-    // enable ratis based SequenceIDGen
-    conf.setBoolean(OZONE_SCM_HA_ENABLE_KEY, true);
```

Comment:
why remove this configuration ?

Reply:
before this commit, a sequenceIdGenerator is build depend on this configuration
`
  if (SCMHAUtils.isSCMHAEnabled(conf)) {
      this.stateManager = new StateManagerHAImpl.Builder()
          .setRatisServer(scmhaManager.getRatisServer())
          .setDBTransactionBuffer(scmhaManager.getDBTransactionBuffer())
          .setSequenceIdTable(sequenceIdTable)
          .build();
    } else {
      this.stateManager = new StateManagerImpl(sequenceIdTable);
    }
`
but now ,  it is no use here and a sequenceIdGenerator is build depend on scmHAManager。

we can also keep it here， it does not matter